Cargo.toml

```diff
@@ -11,14 +11,22 @@ license = "MIT"
 keywords = ["xml", "libxml","xpath", "parser", "parsing", "html"]
 build = "build.rs"
 exclude = [
-  "scripts/*"
+  "scripts/*","benches/*"
 ]
 
+[lib]
+name = "libxml"
+
 [dependencies]
 libc = "0.2"
 
 [build-dependencies]
 pkg-config = "0.3.2"
 
-[lib]
-name = "libxml"
+[dev-dependencies]
+rayon = "1.0.0"
+criterion = "0.2.10"
```

Comment:
the `[dev-dependencies]` header is pretty neat, it allows us to keep very heavy dependencies we'd like to experiment with for benchmarking, while completely ignoring them when `libxml` is used as an external dependency.